internal/provider/resource_tfe_registry_module.go

```diff
@@ -125,11 +150,20 @@ func resourceTFERegistryModuleCreateWithVCS(v interface{}, meta interface{}, d *
 		DisplayIdentifier: tfe.String(vcsRepo["display_identifier"].(string)),
 		OrganizationName:  tfe.String(orgName),
 	}
+	if vcsRepo["branch"].(string) != "" {
```

Comment:
should `create` action take the `tag` field too?

Reply:
The `create` action does not take the `tag` field because that is the default behavior for VCS enabled modules.